.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+cache: pip
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"
+  - "nightly"
+install:
+  - pip install mypy -r requirements.txt
```

Comment:
shouldn't this be `pip3`?

Reply:
it doesn't really matter on Travis. Changed anyway.